model/riscv_addr_checks_common.sail

```diff
@@ -14,11 +14,6 @@
  * extensions would need to define their own functions to override them.
  */
 
-union Ext_ControlAddr_Check ('a : Type) = {
-  Ext_ControlAddr_OK : virtaddr, /* PC value to use for the target of the control operation */
-  Ext_ControlAddr_Error : 'a
-}
-
 union Ext_DataAddr_Check ('a : Type) = {
```

Comment:
I think we could do the same for this (in another PR if you want). @jrtc27 am I right in thinking the latest CHERI never changes the address that's used for memory accesses?